python/lsst/rubintv/analysis/service/commands/db.py

```diff
@@ -204,7 +204,16 @@ def build_contents(self, data_center: DataCenter) -> dict:
         schema_name = f"cdb_{instrument}"
         try:
             database = data_center.schemas[schema_name]
-            result["schema"] = database.schema
+            schema = SchemaInspector(database.schema)
```

Comment:
I would change this to `schema = database.validSchema` (or something like that) and implement the rest of this in the `ConsDbSchema` class itself.